website/docs/software_partners/explanations/life_cycle_of_an_onboarding_session/index.md

```diff
@@ -58,12 +58,24 @@ stateDiagram-v2
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Abandoned                    | Fired after two weeks if the onboarding session has not been completed. Will trigger abandonment behavior then wait for the results. This for example will be to start the stapling and defaulting behavior |
+| Archived                     | Fired after four weeks regardless of what state the onboarding session is. Any PII about the employee is removed at this step.                                                                              |
+| Duplicate onboarding session | If another onboarding session is created while a previous one exists for the same workflow and employee, we immediately archive the existing onboarding session                                             |
+
+## Outcomes
+
+Each module in the workflow will record an `outcome` key which details the 'outcome' of the user interacting with that module. Outcomes for most modules usually fall into either a 'complete' or 'incomplete'. Take for example the bank accounts module, if the employee enters their bank account details and clicks next then the outcome of the bank account module will be that it was 'complete'.
+
+One special exception to the complete / incomplete outcome is the `onboarding_session_super_selection` module as this has more possible end states. These outcomes are recorded in the following table along with some possible actions you may want to take in your system:
+
+| Outcome                        | Reason                                                                                                                                                                                              | Possible actions                                                            |
+| ------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| complete                       | The user completed all steps required to provide details for payment to their nominated super fund.                                                                                                 | Submit the employees details to payroll                                     |
+| complete_without_member_number | The user did not complete the form, we were unable to find their stapled superfund and we did not receive a response from sending a request (MRR) to join them to the employers default super fund. | Notify the employer to collect superfund details directly from the employee |
```

Comment:
Should the possible actions mention something about informing the employer to join the member to the employer's default super fund?

Reply:
Good pickup - I've made the employer actions more explicit.